scanner/src/main/kotlin/scanners/fossid/FossId.kt

```diff
@@ -319,6 +319,11 @@ class FossId(
             val filteredPackages = packages
                 .filter { packageNamespaceFilter.isEmpty() || it.id.namespace == packageNamespaceFilter }
                 .filter { packageAuthorsFilter.isEmpty() || packageAuthorsFilter in it.authors }
+                .onEach {
+                    if (it.vcs.path.isNotEmpty()) {
+                        log.warn { "Ignoring package with url ${it.vcs.url} with non-null path ${it.vcs.path}" }
+                    }
+                }.filter { it.vcsProcessed.path.isEmpty() }
```

Comment:
why do you use vcsProcessed here and vcs above?

Reply:
I was wrong, thanks.